report_footer_html/models/ir_actions_report.py

```diff
@@ -8,7 +8,7 @@ class IrActionsReport(models.Model):
     _inherit = "ir.actions.report"
 
     footer_html = fields.Html(
-        string="Footer HTML",
+        string="Footer",
```

Comment:
I guess this change is due to the warning in the logs stating that the label is the same that the string. But you just need to remove the string parameter

Reply:
This change was made to match the string that Odoo uses natively, as @Shide proposed to me.

<img width="518" height="97" alt="image" src="https://github.com/user-attachments/assets/7fdd82bf-154a-4ea1-8e65-6e2661922a62" />

Reply:
What about changing it to "Extra footer" as it describes better how it really behaves?